Source/Management.Web/WebPack/devServer.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+const fileTypes = /\.(js|css|html|png|jpg|jpeg|gif)$/;
+
+module.exports = (basePath, port) => {
+    const actualPort = process.env.port || port || 9000;
+
+    return {
+        historyApiFallback: { index: basePath },
+        host: '0.0.0.0',
+        port: actualPort,
+        publicPath: basePath,
+        contentBase: process.cwd(),
+        proxy: {
+            '/api': 'http://localhost:3000',
+            '/graphql': 'http://localhost:3000',
```

Comment:
Isn't the port 5000?

Reply:
Its overridden in the WebPack.config - this whole folder is a copy/paste from Vanir.